seqr/views/apis/auth_api.py

```diff
@@ -35,6 +42,111 @@ def login_view(request):
 
     return create_json_response({'success': True})
 
+@csrf_exempt
+def login_google(request):
+  # Create flow instance to manage the OAuth 2.0 Authorization Grant Flow steps.
+  flow = google_auth_oauthlib.flow.Flow.from_client_config(
```

Comment:
It can be set up once and reused. But, I'm sure about saving: storage of a global `flow` vs. computation of a dynamical created `flow` when there is a request to log in.